sbi/utils/sbiutils.py

```diff
@@ -1006,3 +1018,43 @@ def seed_all_backends(seed: Optional[Union[int, Tensor]] = None) -> None:
     torch.cuda.manual_seed(seed)
     torch.backends.cudnn.deterministic = True  # type: ignore
     torch.backends.cudnn.benchmark = False  # type: ignore
+
+
+def warn_if_deprecated(
+    method: Callable, locals_dict: Dict[str, Any], deprecated_keys: Set
```

Comment:
I suggest renaming `locals_dict` to `local_args` because type will say that it is a `Dict` and `local_args` reads better then.